frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -1,8 +1,8 @@
-import 'dart:ffi';
 import 'dart:typed_data';
 
 import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
-import 'package:flutter_rust_bridge_example/bridge_generated.dart';
+import 'bridge_definitions.dart';
+import 'bridge_generated.dart';
```

Comment:
TODO: add web tests

(just make it a review comment, such that github will prevent us from merging the PR accidentically before resolving this comment, so we will not forget it)

Reply:
Web tests can now be run with `just test-pure-web` 🎉